modules/installation-cloudformation-worker.adoc

```diff
@@ -17,3 +16,24 @@ that you need for your {product-title} cluster.
 include::https://raw.githubusercontent.com/openshift/installer/release-4.18/upi/aws/cloudformation/06_cluster_worker_node.yaml[]
 ----
 ====
+[IMPORTANT]
+====
+When you use the CloudFormation template for the master machines, it provisions all three master machines with a single stack. However, when you use the CloudFormation template to deploy the worker machines, you must create the number of stacks based on the number that you defined in the `install-config.yaml` file. Each stack is provisioned one time only for each machine. To provision a new worker machine, you must change the stack name.
+====
+
+[source,terminal]
+----
+$ aws cloudformation create-stack --stack-name <name> <1>
+     --template-body file://<template>.yaml \ <2>
+     --parameters file://<parameters>.json <3>
+----
+<1> `<name>` is the name for the CloudFormation stack, such as `cluster-worker-1`. You need the name of this stack if you remove the cluster.
+<2> `<template>` is the relative path to and name of the CloudFormation template YAML file that you saved.
+<3> `<parameters>` is the relative path to and name of the CloudFormation
+parameters JSON file.
```

Comment:
As per guideline[1], we avoid starting any sentence with lowercase. So consider updating all call out descriptions with something like
Option 1:  -- "Replace `<name>` with the name of...." 
Option 2: -- "The `<name> parameter....."
Option 3: -- "Defines the name of the....."

[1] https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#general-text

> Do not start a sentence with a lowercase term; rewrite the sentence.